mainhandler/vulnscan.go

```diff
@@ -287,6 +287,15 @@ func (actionHandler *ActionHandler) scanWorkload(ctx context.Context, sessionObj
 		logger.L().Info(fmt.Sprintf("workload %s has no podSpec, skipping", actionHandler.wlid))
 		return nil
 	}
+
+	// get pod instanceID
+	instancesID, err := instanceidhandler.GenerateInstanceIDFromPod(pod)
```

Comment:
instanceIDs ? I'm not sure about plurals for composed words.

Reply:
Resolved in [this](https://github.com/kubescape/operator/pull/99/commits/7f88b33efd8b0c5d9c0d7d00961022b607d96311) commit